cfgov/unprocessed/css/misc.less

```diff
@@ -726,6 +726,29 @@
     opacity: 0;
 }
 
+/* topdoc
+  name: Download icons
+  family: cfgov-misc
+  notes:
+    - "Adds download icons to certain links"
+  tags:
+    - cfgov-misc
+*/
+
+
+a[href$='.pdf']:after,
+a[href$='.doc']:after,
+a[href$='.docx']:after,
+a[href$='.xls']:after,
+a[href$='.xlsx']:after,
+a[href$='.csv']:after,
+a[href$='.zip']:after {
+    .cf-icon();
+    .u-inline-block();
```

Comment:
At this point this could be straight up `display: inline-block`. The utility class just adds a hack for IE7 and below https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-utilities.less#L113